pkg/storage/storageclass.go

```diff
@@ -101,9 +103,7 @@ func (builder *ClassBuilder) WithVolumeBindingMode(
 		glog.V(100).Infof("The bindingMode of the storageclass is empty")
 
 		builder.errorMsg = "storageclass 'bindingMode' cannot be empty"
-	}
 
-	if builder.errorMsg != "" {
```

Comment:
If you remove this condition we add invalid bindingMode to builder. Please revert it back

Reply:
Same here, the diff on Github makes it look confusing for some reason.  The actual code won't allow an invalid bindingMode though.

![image](https://github.com/user-attachments/assets/8a5a1083-f273-4958-aaf7-f2c08ce97778)